src/Components/TasksList.js

```diff
@@ -0,0 +1,60 @@
+
+import React, { useState } from "react";
+
+const TasksList = ({ id, listTodoToday, listTodoTomorrow, listTodoUpcoming }) => {
+  const [checkedItems, setCheckedItems] = useState([]);
+
+  const handleCheckboxChange = (event, item) => {
+    const isChecked = event.target.checked;
+    if (isChecked) {
+      setCheckedItems([...checkedItems, item]);
+    } else {
+      setCheckedItems(checkedItems.filter((checkedItem) => checkedItem !== item));
+    }
+  };
+
+  const isItemChecked = (item) => {
+    return checkedItems.includes(item);
+  };
+
+  return (
+    <div className="tasksList">
+      {id === "1"
```

Comment:
The condition "id === 1" is not easily understandable to me. Let's revise it to a clearer representation. How about using "timeline === 'Today'" as an example?